app/models/test_suite.py

```diff
@@ -112,55 +142,71 @@ def created_on(self):
     @created_on.setter
     def created_on(self, value):
         """Set the creation date of this test suite."""
+        if self._created_on:
+            raise AttributeError("Creation date already set")
         self._created_on = value
 
     def to_dict(self):
-        test_suite = {
+        return {
             models.ARCHITECTURE_KEY: self.arch,
             models.BOARD_INSTANCE_KEY: self.board_instance,
             models.BOARD_KEY: self.board,
-            models.BOOT_ID_KEY: self.boot_id,
+            models.BOOT_LOG_KEY: self.boot_log,
+            models.BOOT_LOG_HTML_KEY: self.boot_log_html,
+            models.BOOT_RESULT_DESC_KEY: self.boot_result_description,
             models.BUILD_ID_KEY: self.build_id,
+            models.COMPILER_KEY: self.compiler,
+            models.COMPILER_VERSION_FULL_KEY: self.compiler_version_full,
+            models.COMPILER_VERSION_KEY: self.compiler_version,
+            models.CROSS_COMPILE_KEY: self.cross_compile,
             models.CREATED_KEY: self.created_on,
             models.DEFCONFIG_FULL_KEY: self.defconfig_full or self.defconfig,
             models.DEFCONFIG_KEY: self.defconfig,
             models.DEFINITION_URI_KEY: self.definition_uri,
+            models.DEVICE_TYPE_KEY: self.device_type,
+            models.DTB_KEY: self.dtb,
+            models.DTB_ADDR_KEY: self.dtb_addr,
+            models.ENDIANNESS_KEY: self.endian,
+            models.FILE_SERVER_RESOURCE_KEY: self.file_server_resource,
+            models.FILE_SERVER_URL_KEY: self.file_server_url,
             models.GIT_BRANCH_KEY: self.git_branch,
+            models.GIT_COMMIT_KEY: self.git_commit,
```

Comment:
Shouldn't we align on this GIT_COMMIT_KEY vs VCS_COMMIT_KEY (line 194) ? Seems like a duplicate

Reply:
Yes, it is a duplicate.  I left both to be compatible with current boot entries and also preserve the fields currently used in test entries.

Do we really need `VCS_COMMIT_KEY`?

Reply:
I don't think so, I've never seen it used, or even set...

Reply:
OK then I think we can just drop `VCS_COMMIT_KEY` from the test entries.  I'll add another patch for that.